.github/workflows/bubblesub.yml

```diff
@@ -1,6 +1,6 @@
 name: bubblesub
 
-on: [push, pull_request]
+on: [pull_request]
```

Comment:
Why have we removed the `push` event?

Reply:
IMO `pull_request` is enough, no need to run the jobs twice.

Reply:
The main problem is when we decide to push directly on `master`, in that case any CI check will be triggered. I don't agree with removing this event

Reply:
In that case leaving it as `push` is enough since it's also triggered for PRs.